crates/mipsy_web/src/pages/main/app.rs

```diff
@@ -42,7 +42,6 @@ pub fn render_app() -> Html {
     let state: UseStateHandle<State> = use_state_eq(|| State::NoFile);
 
     let worker = Rc::new(RefCell::new(None));
-    let on_content_change_closure_handle: UseStateHandle<bool> = use_state_eq(|| false);
```

Comment:
this,,, also causes issues and removing it fixes a lot of them
i think I added it to make sure only one callback was ever made and leaked, but i did a memory profile after lots of swapping and surprisingly the removed version was lesser memory!